bench/workgen/workgen.cxx

```diff
@@ -275,11 +275,42 @@ WorkloadRunner::start_table_idle_cycle(WT_CONNECTION *conn)
 int
 WorkloadRunner::start_dynamic_table_mgmt(WT_CONNECTION *conn)
 {
-    while (!stopping) {
-        std::cout << "Exec dynamic table mgmt..";
-        sleep(1);
+    WT_SESSION *session;
+    if (conn->open_session(conn, nullptr, nullptr, &session) != 0) {
+        THROW("Error opening a session.");
     }
 
+    for (uint64_t create_count = 0; !stopping; ++create_count) {
+        char uri[BUF_SIZE];
+        snprintf(uri, BUF_SIZE, "table:test_dynamic_%" PRIu64, create_count);
+
+        /* Create a table. */
+        int ret;
+        if ((ret = session->create(session, uri, "key_format=S,value_format=S")) != 0) {
+            if (ret == EBUSY)
+                continue;
```

Comment:
This will trigger the next iteration which means incrementing `creates ` and `total_creates ` by 1 while nothing has been done, is it intended?

Reply:
This is intentional for now. I will change this when I generate random tables and not sequential ones.